### PR TITLE
Move the DB schema out of the code in the case of MySQL

### PR DESCRIPTION
**Pull request by codefromthecrypt**
Polishes @jfeltesse-mdsol's pull request, by documenting how to install it, and erring if someone expects auto-install.

``` bash
$ mysql -uroot -e "create database if not exists zipkin"
# install the schema and indexes
$ mysql -uroot -Dzipkin < zipkin-anormdb/src/main/resources/mysql.sql
$ MYSQL_USER=root ./bin/collector mysql
...
```

Fixes #704

### CODE REVIEW

File: zipkin-anormdb/src/main/resources/mysql.sql [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  dlid BIGINT NOT NULL,
+  parent VARCHAR(255) NOT NULL,
+  child VARCHAR(255) NOT NULL,
+  m0 BIGINT NOT NULL,
+  m1 DOUBLE PRECISION NOT NULL,
+  m2 DOUBLE PRECISION NOT NULL,
+  m3 DOUBLE PRECISION NOT NULL,
+  m4 DOUBLE PRECISION NOT NULL
+);
+ALTER TABLE zipkin_spans ADD PRIMARY KEY(span_id);
```
Comment (jfeltesse-mdsol) [inline zipkin-anormdb/src/main/resources/mysql.sql]: nitpicking here but there should be an empty line above and, having said that, if we're going to have the schema and index defs done at once maybe having the index defs close to the table they change would be easier to understand?

Comment (codefromthecrypt) [inline zipkin-anormdb/src/main/resources/mysql.sql]: sure thing

**Comment by jfeltesse-mdsol** [comment]
cool stuff! :100:

File: zipkin-anormdb/src/main/resources/mysql.sql (hunk 1 above)
Comment (codefromthecrypt) [inline zipkin-anormdb/src/main/resources/mysql.sql]: done

**Comment by jfeltesse-mdsol** [comment]
:ok_woman:

**Pull request merged**